Tools/stringbench/stringbench.py

```diff
@@ -730,7 +730,7 @@ def join_alphabet_list_five(STR):
     for x in _RANGE_1000:
         sep_join(s2)
 
-@bench('"A".join(["Bob"]*100))',
+@bench('"A".join(["Bob"]*100)',
```

Comment:
Line 742 below needs the same correction.

I considered whether the 'mistake' might be intentional and checked the definition of bench.  The first argument is attached to the function f as f.comment.  So the error has no possible functional use and should be corrected.